crates/rattler_networking/src/authentication_storage/storage.rs

```diff
@@ -140,6 +162,10 @@ impl AuthenticationStorage {
             cache.remove(host);
         }
 
+        if self.force_fallback_storage {
```

Comment:
At the end we do also delete the password from any fallback storage. So maybe this is not needed here?